__tests__/user/user.mutation.test.js

```diff
@@ -766,3 +786,136 @@ describe('User`s mutation restictions tests', () => {
     expect(result.graphQLErrors[0].message).toEqual('USER_NOT_AUTHORIZED');
   });
 });
+
+describe("Creating users with custom roles",() => {
```

Comment:
Please, add negative checks for cases:
1) `registerAdmin`:
   - invalid email provided and proper error messages are returned
   - invalid role provided and proper error messages are returned
   - admin with provided email already exists
2) `completeAdminRegister`:
   - invalid `firstName`, `lastName`, `password` provided and proper error messages are returned
   - invalid token is provided and proper error message is returned
   - invalid token is provided and a proper error message is returned
   - valid token is provided but the user is not found and a proper error message is returned

Reply:
Also, I think it is good to add the query that allows verifying that the token from an email is valid. It is useful when you can check is token valid before rendering the form for inputting missing admin data. Also, please, add tests for this query in the same fashion as this one

Reply:
l did it

Reply:
Also, please, group all of your tests in `describe` statements per operation. For example, you should have `describe('registerAdmin')` and inside this describe you have all tests both negative and positive for this particular operation.

Reply:
Also, please, restrict the visibility of global variables that are defined below. If some variable is used only for some particular operation, it should defined only inside the `describe` statement for that particular operation